test/parallel/test-worker-relative-path-double-dot.js

```diff
@@ -0,0 +1,21 @@
+// Flags: --experimental-worker
+'use strict';
+const path = require('path');
+const assert = require('assert');
+const common = require('../common');
+const { Worker, isMainThread, parentPort } = require('worker_threads');
+
+if (isMainThread) {
+  const cwdName = path.relative('../', '.');
+  const relativePath = path.relative('.', __filename);
+  const w = new Worker(path.join('..', cwdName, relativePath));
+  w.on('message', common.mustCall((message) => {
+    assert.strictEqual(message, 'Hello, world!');
+  }));
+} else {
+  setImmediate(() => {
+    process.nextTick(() => {
```

Comment:
why the `nextTick` here? Seems a bit pointless.

Reply:
Actually it was simply copied from [test-worker.js](https://github.com/nodejs/node/blob/master/test/parallel/test-worker.js)

Reply:
For context, inside that test the purpose is to make sure that both libuv bindings + process.nextTick() work – we don’t need that here, and I probably should have added a comment to the other test about it ;)

Reply:
Got it, thanks.